ex07/ex07.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+if [[ -z "$1" ]] || [[ "$1" -eq 0 ]]; then
+	exit 1
+fi
+res=$(head "-$1" resourses/surnames.txt | grep -v 'Q-Chem' | sed "s/\.//g ; s/\-//g" | cat)
+if [[ $res ]]; then
+	echo "$res"
+else
+	echo ""
+fi
```

Comment:
Test on 0 lines returns error and Test on 2 lines return empty string tests are failed